src/seagap/configs/main.py

```diff
@@ -118,3 +119,26 @@ class Configuration(BaseConfiguration):
     # TODO: Separate settings out to core plugin
     solver: Optional[Solver] = Field(None, description="Solver configurations")
     output: OutputPath
+
+
+def load_configuration(config_yaml: Optional[str] = None):
+    try:
+        config = Configuration()
+    except ValidationError:
+        warnings.warn(
+            "Loading attempt failed, trying to load configuration from file path given."
+        )
+        if config_yaml is None:
```

Comment:
to avoid duplicating the exception code:
```suggestion
        if config_yaml is None or not Path(config_yaml).exists():
```